01_login.php

```diff
@@ -36,149 +36,165 @@
 								</div>
+							</div>
 
-								<p><button class="w3-button w3-section w3-white w3-ripple w3-hover-orange"  style="margin-left: 160px;"> Log in </button></p>
-
-
-								<hr>
-								<!-- register form; for new users -->
-								<div style="margin-left: 139px;">
-									<p1>Don't have an account?</p1>
+							<div class="w3-row w3-section">
+								<div class="w3-col" style="width:50px"><span class="w3-xxlarge material-icons">lock</span></div>
+								<div class="w3-rest">
+									<input for='login' class="w3-input w3-border" type = "password" name = "password" placeholder="Password" style="width:100%" required/>
 								</div>
+							</div>
+							<div class="w3-center">
+								<button class="w3-button w3-section w3-green w3-ripple w3-hover-grey"> Log in </button>
+							</div>
 
-								<br>
-								<div style="margin-left: 100px;">
-									<button onclick="document.getElementById('id01').style.display='block'" class="w3-button w3-white w3-ripple w3-hover-orange">SIGN UP FOR SOUNDWAVE</button> 
-								</div>
+							<!-- creates a line to seperate both login form / register -->
+							<hr>
+							<div class="w3-center">
+								<p1>Don't have an account?</p1>
+							</div>
 
+							<br>
+							<div class="w3-center">
+								<button onclick="document.getElementById('id01').style.display='block'" class="w3-button w3-white w3-ripple w3-hover-grey">SIGN UP FOR SOUNDWAVE</button> 
+							</div>
 
-								<h3 class = "white"><?php echo $error; ?></h3>
-								<br>
-							</form>
-						</div>
+							<h3 class = "white"><?php echo $error; ?></h3>
+							<br>
+						</form>
 					</div>
 				</div>
-				
+			</div>
+			
+			<!-- register form - modal pop up -->
+			<div id="id01" class="w3-modal">
+				<div class="w3-modal-content w3-card-4 w3-animate-zoom w3-white" style="max-width:600px">
 
-				<div id="id01" class="w3-modal">
-					<div class="w3-modal-content w3-card-4 w3-animate-zoom w3-black" style="max-width:600px">
+					<div class="w3-center"><br>
+						<span onclick="document.getElementById('id01').style.display='none'" class="w3-button w3-xlarge w3-transparent w3-display-topright" title="Close Modal">×</span>
+					</div>
 
-						<div class="w3-center"><br>
-							<span onclick="document.getElementById('id01').style.display='none'" class="w3-button w3-xlarge w3-transparent w3-display-topright" title="Close Modal">×</span>
+					<form class="w3-container w3-white" action="/action_page.php">
+						<div class="w3-section">
+							<label for = 'login'>Username:</label>
+							<input type = "text" name = "username" placeholder="Enter user name" style="width:100%" required/><br />
+							<label for = 'login'>Password:</label>
+							<input type = "password" name = "password" placeholder="Enter user password" style="width:100%" required/><br />
+							<a href="01_loginV2.php" class="w3-button w3-block w3-green w3-section w3-padding" type="submit">Register </a>
```

Comment:
green button to add another colour to the palette: green, since green and orange work well together